bdk_cli_lib/src/lib.rs

```diff
@@ -19,14 +19,15 @@ use bdk_chain::{
 use bdk_coin_select::{coin_select_bnb, CoinSelector, CoinSelectorOpt, WeightedValue};
 pub use clap;
 use clap::{Parser, Subcommand};
+pub use log;
```

Comment:
Not sure if we need this. I think CLI should try and avoid using logging unless we're going into a kind of `daemon` mode. I think env_logger is a good idea though but I don't see it being initialized anywhere yet.